embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/injection/InitModule.kt

```diff
@@ -81,10 +82,13 @@ internal class InitModuleImpl(
         SpansSinkImpl()
     }
 
+    private val exporter = CompositeSpanExporter()
+
     private val openTelemetrySdk: OpenTelemetrySdk by singleton {
+        exporter.add(EmbraceSpanExporter(spansSink))
         OpenTelemetrySdk(
             openTelemetryClock = openTelemetryClock,
-            spanProcessor = EmbraceSpanProcessor(EmbraceSpanExporter(spansSink))
```

Comment:
Hmm. The reason you can't use the default composite one is because we are initializing this too early, before the app can add the exporter instance? If this were happening later, I can imagine a list of exporters passed into `InitModuleImpl` and we just call `SpanExporter.composite()` here

Reply:
Exactly. We are exporting the `emb-startup-moment` span, though.